scripts/gefs_atmos_f0306.sh

```diff
@@ -0,0 +1,192 @@
+    ${WGRIB2} "${infile}" | grep ":HCDC:high cloud layer:0-6" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":MCDC:middle cloud layer:0-6" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":LCDC:low cloud layer:0-6" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":GFLUX:surface" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":U-GWD:surface" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":V-GWD:surface" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":APCP:surface" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":ACPCP:surface" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep ":NCPCP:surface" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "TMP:middle cloud top" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "TMP:low cloud top" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "TMP:high cloud top" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "PRES:high cloud top" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "PRES:middle cloud top" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "PRES:low cloud top" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "PRES:high cloud bottom" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "PRES:middle cloud bottom" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "PRES:low cloud bottom" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "CWORK" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "DUVB" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "CDUVB" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "TMAX" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+    ${WGRIB2} "${infile}" | grep "TMIN" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null
+
+   else
+    echo "${infile} does not exist"
+   fi
+
+ export exec_dir=$EXECacc
+ export sorc_dir=$SORCacc
+ export sorc_name=gefs_6h_ave_1mem
+
+#   cd $DATA
+
+   $exec_dir/$sorc_name >sorc_name.exe.out
+   cat sorc_name.exe.out
```

Comment:
@WalterKolczynski-NOAA, @HongGuan-NOAA has source code for this task that has not been added to this PR. Should her source code be added to gfs-utils in a similar manner as the other GEFS-specific programs (e.g. [gfs-utils PR #52](https://github.com/NOAA-EMC/gfs-utils/pull/52))?

Reply:
Yes. I thought I had already talked with Hong about opening a PR.